### ISSUE TITLE
"Nice name" in the simulators list?

### ISSUE THREAD

**Issue by sole** [mozilla/fireplay-sublime#70]
I wonder if it's possible to show a nicer name in this list:
![screenshot 2014-08-13 17 10 31](https://cloud.githubusercontent.com/assets/5609/3908537/747a9758-2304-11e4-9d46-b0871304721c.png)

**Comment by nicola** [comment]
Yes we can! The regular expression to get the number is already in the code.
